query/language/define.feature

```diff
@@ -394,6 +427,38 @@ Feature: TypeQL Define Query
       | label:pet-ownership |
 
 
+  Scenario: new relation type declaration should contain kind
+    Then typeql schema query; fails
+      """
+      define pet-ownership;
+      """
+
+
+  Scenario: new relation types can be defined in multiple steps within one query
+    When typeql schema query
+      """
+      define
+      pet-ownership relates pet-owner;
+      pet-ownership relates owned-pet;
+      pet-ownership @abstract;
+      pet-ownership;
```

Comment:
Do we really want to commit to supporting this empty define statement?

Reply:
If we decide to reject it, we can rewrite the test. The worst case is, to my mind, when we are not even aware if it's possible or not, thus not covered by tests.  
If this case doesn't exist, we may not remember to write a negative-case test when we change the behavior.